lib/reporter/cli.js

```diff
@@ -90,6 +95,23 @@ class Cli extends Base {
       });
     }
 
+    runner.on('suite end', suite => {
+      let skippedCount = 0;
+      const grep = require('../container').mocha().options.grep;
```

Comment:
I think you can get mocha options from runner here.
I would not recommend use circular reference like this: mocha -> cli -> container -> mocha